core/miner/storage_fsm/impl/precommit_batcher_impl.hpp

```diff
@@ -13,19 +13,23 @@
 #include <mutex>
 #include "api/full_node/node_api.hpp"
 #include "primitives/address/address.hpp"
+#include "miner/storage_fsm/types.hpp"
 
 namespace fc::mining {
   using api::FullNodeApi;
   using libp2p::basic::Scheduler;
   using primitives::SectorNumber;
   using primitives::address::Address;
+  using types::FeeConfig;
 
   class PreCommitBatcherImpl : public PreCommitBatcher {
    public:
     PreCommitBatcherImpl(const std::chrono::milliseconds &max_time,
                          std::shared_ptr<FullNodeApi> api,
                          const Address &miner_address,
-                         const std::shared_ptr<Scheduler> &scheduler);
+                         const std::shared_ptr<Scheduler> &scheduler,
+                         const std::function<outcome::result<Address>(MinerInfo miner_info, TokenAmount deposit, TokenAmount good_funds)>&,
```

Comment:
Reuse with `using ALIAS = LONG_TYPE_NAME...;`

Reply:
Also prefer const references for `MinerInfo` and `TokenAmount` immutable args

Reply:
fixed